test_scripts/MobileProjection/Phase2/005_audioStreamingState_111-114.lua

```diff
@@ -0,0 +1,64 @@
+---------------------------------------------------------------------------------------------------
+-- Issue:
```

Comment:
Please update checklist according to updated cases number. Checklist contains 18 cases.

Reply:
@GetmanetsIrina Test cases for 'COMMUNICATION' app is removed since this type of app can't stream